companies/dbregio.json

```diff
@@ -0,0 +1,27 @@
+{
+    "slug": "dbregio",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "DB Regio AG",
+    "runs": [
+        "bahn.de"
```

Comment:
[https://www.bahn.de/datenschutz](https://www.bahn.de/datenschutz)
-> "Wer ist verantwortlich für die Datenerhebung und -verarbeitung?" lists multiple companies responsible for bahn.de.
-> "Welche Rechte haben Nutzerinnen und Nutzer von bahn.de/bahn.com?" states that DB Vertrieb GmbH should be contacted. 
What should be done here?

Reply:
Hm, we currently have a record for DB Vertrieb GmbH (https://www.datarequests.org/company/deutsche-bahn) that lists all those companies and bahn.de under `runs`.

According to "Welche Daten verarbeiten wir und wie und warum verarbeiten wir Ihre Daten?", DB Vertrieb GmbH is involved in all processing activities. And as you said, the privacy also tells you to contact them and even says "Sofern Sie uns anschreiben, Ihr Anliegen aber den Verantwortungsbereich des von Ihnen genutzten Dienstes betrifft, leiten wir dieses an ihn weiter."

I can also confirm from a prior request to DB Vertrieb GmbH, that the reply includes the data on everything.

So, I think we don't need this record. But we should update the DB Vertrieb GmbH one as at least the address of that one is outdated. What do you think?

Reply:
Agreed. Both options aren't perfect, but if a request to DB Vertrieb GmbH reports back data from all related companies, this is IMHO definitely the better option.

Reply:
I have updated the contact details in b81fa4facb5e7b176bc1c1e4c2fbade414de5091.